lib/foreman_maintain/cli.rb

```diff
@@ -25,6 +26,7 @@ class MainCommand < Base
       subcommand 'packages', 'Lock/Unlock package protection, install, update', PackagesCommand
       subcommand 'advanced', 'Advanced tools for server maintenance', AdvancedCommand
       subcommand 'content', 'Content related commands', ContentCommand
+      subcommand 'plugin', 'Control features of your server', PluginCommand
```

Comment:
Is this actually a good name?
Should we call it something else? `feature`?

Reply:
Ah, the naming problem. It always comes up. This is technically the first time we deal with such a thing in downstream. Before this it was always a "fixed set".

What came close was the foreman_docker removal and optional installation of Kubevirt. However, that wasn't handled via explicit maintain commands.

`feature` may conflict in naming and I think `plugin` feels natural. However, I would suggest
```suggestion
      subcommand 'plugin', 'Manage optional plugins on your server', PluginCommand
```